cmd/geth/config.go

```diff
@@ -115,12 +129,28 @@ func defaultNodeConfig() node.Config {
 	return cfg
 }
 
+func defaultMetricConfig(ctx *cli.Context) metricConfig {
```

Comment:
Perhaps this should be `metricsFromCliArgs`.. When I saw `defaultMetricConfig` being used before, I intuitively though that it just initialized some defaults, and didn't grokk that it actually read the cli arguments

Reply:
Yeah, that's the thing with this strange logic, we need to read the metrics twice. 
First we need to read the "default" metrics and apply them s.th. if we don't override them in the load config, they are set to the default values.
Then, after reading the config, we need to read the metrics again (this time only the specially set metrics) to overwrite the config file configs with the configs from cli.

So basically we do:
- read config from cli regardless of whether they are set (reads defaults)
- read config from file (overwrites the default configs if applicable)
- read only the set configs from cli (overwrites the file configs)

I'll change the naming!